Automattic/wp-calypso
# Checkout: Remove sidebar AB test code

Now that the "Included with your Purchase" and "Upsell Nudge" a/b tests have completed, this PR removes their associated explat code.

Related to #84891 and #84890 

## Proposed Changes

* This PR will remove the experiment hook files along with the calling code throughout checkout

## Testing Instructions

* Add products to your cart and go to Checkout - including a one year plan
* Ensure that the 'Included with your purchase' featured list and the Two Year Plan upsell nudge both appear in the checkout sidebar
* This should work regardless of which a/b group you have been assigned in the past, since the explat code would be removed and no longer care for these group assignments
